pkgs/record_use/lib/src/proto/usages_read.pbenum.dart

```diff
@@ -0,0 +1,10 @@
+//
+//  Generated code. Do not modify.
+//  source: usages_read.proto
+//
+// @dart = 2.12
```

Comment:
Note to self: `2.12` is now considered an old version of Dart; we should update the proto generator to use at leat `3.0`.